docs/book/src/topics/consuming-existing-aws-infrastructure.md

```diff
@@ -34,6 +34,10 @@ Finally, if the controller manager isn't started with the `--configure-cloud-rou
 Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object. Specifically, you will need to add an entry with the VPC ID and the IDs of all applicable subnets into the `networkSpec` field. Here is an example:
 
 ```yaml
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
```

Comment:
For EKS based clusters it lives on **AWSManagedControlPlane** but for EC2 based clusters it lives on **AWSCluster**. For each of these kinds it is in the spec.

So i think we should we should either:
1. Change this part `Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object`
2. Have a separate example or section for each cluster type